float64 timestamp
string vehicle_id
string status